pulp_container/app/authorization.py

```diff
@@ -82,13 +82,12 @@ def generate_token(self):
         )
 
         with open(settings.PRIVATE_KEY_PATH, "rb") as private_key:
-            binary_token = jwt.encode(
```

Comment:
since you're touching the application code, does it make sense to create a misc entry with the explanation?

Reply:
🤷🏻‍♂️